### PR TITLE
ngViewer - Initial Commit

### PR DESCRIPTION
**Pull request by Selameab**

### CODE REVIEW

File: ngViewer/index.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html ng-app="ngViewer">
+<head>
+
+    <meta charset="utf-8">
+
+    <!--CSS-->
+    <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css"/>
+    <link rel="stylesheet" type="text/css" href="css/bootstrap-theme.min.css"/>
+    <link rel="stylesheet" type="text/css" href="font-awesome/css/font-awesome.min.css"/>
```
Comment (AmeBel) [inline ngViewer/index.html]: what are the different `font-awesome/*` files for if `font-awesome/css/font-awesome.min.css` is the only one used.

**Comment by Selameab** [comment]
I didn't notice that the non-minified version was there. Anyways it's gone now...

File: ngViewer/index.html (hunk 1 above)
Comment (Selameab) [inline ngViewer/index.html]: I didn't notice that the non-minified version was there. Anyways it's gone now...

Comment (AmeBel) [inline ngViewer/index.html]: are the different `font-awesome/*` directories required?

Comment (Selameab) [inline ngViewer/index.html]: They are not necessary for the app to work. But they might come in handy if we decide to modify the css.

Comment (AmeBel) [inline ngViewer/index.html]: Please don't vendor in unused source code.

How can grunt.js (or some other task-runner) be used with package.json file for setting up the environment? I am asking b/c it isn't a good practice to have dependencies committed in version-control.

Comment (linas) [inline ngViewer/index.html]: Amen is right; copying in large quantities of code from external sources is a bad idea. It always comes back to bite.

Comment (Selameab) [inline ngViewer/index.html]: Accepted - I'll cleanup all unused code and include a task runner.

**Pull request closed without merging**